### ISSUE TITLE
API-Config file with spaces lead to FileNotFoundException

### ISSUE THREAD

**Issue by RadhaR4** [Axway-API-Management-Plus/apimanager-swagger-promote#135]
```
./run-swagger-import.sh -a "/opt/swaggers/AS Auth Inquiry" -c "/opt/config/AS Auth Inquiry config.json" -h localhost -u apiadmin -p password
```

```
com.axway.apim.lib.AppException: Cant parse JSON-Config file(s)
        at com.axway.apim.swagger.APIImportConfigAdapter.<init>(APIImportConfigAdapter.java:128)
        at com.axway.apim.App.run(App.java:179)
        at com.axway.apim.App.main(App.java:44)
Caused by: java.nio.file.NoSuchFileException: /opt/config/AS
        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
        at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
        at java.nio.file.Files.newByteChannel(Files.java:361)
        at java.nio.file.Files.newByteChannel(Files.java:407)
        at java.nio.file.Files.readAllBytes(Files.java:3152)
        at com.axway.apim.swagger.APIImportConfigAdapter.substitueVariables(APIImportConfigAdapter.java:141)
        at com.axway.apim.swagger.APIImportConfigAdapter.<init>(APIImportConfigAdapter.java:113)
```

**Comment by cwiechmann** [comment]
Hi @RadhaR4, you have reported that even with the fix I made above the error still occurs on your side. The difference is that you are using a Linux-Environment and I have tested on Windows. 

I was able to reproduce the problem on Linux as well and found the root cause in the Shell-Script: run-swagger-import.sh transferring the parameters to the Java-Application.

I changed the following line from:
`"$JAVA_HOME/bin/java" -Xms64m -Xmx256m -classpath "$CP" com.axway.apim.App $*`
to:
`"$JAVA_HOME/bin/java" -Xms64m -Xmx256m -classpath "$CP" com.axway.apim.App "${@}"`

Please do the same change and let me know, if it's working.

**Comment by cwiechmann** [comment]
Has been fixed/enhanced in release 1.6.2.
